syntax = "proto3";

package grpc_signer;

service SignerCommunication {
    rpc pullData (SignerDataRequest) returns (SignerDataReply) {}
}
message SignerDataRequest {
    int32 check = 1;
}
message SignerDataReply {
    bytes message = 1;
}
